.github/workflows/ci.yml

```diff
@@ -127,11 +127,23 @@ jobs:
       # String to describe the overall check
       commit-status-description: ${{ steps.commit-status-args.outputs.description }}
     steps:
+      - name: Check commenter permissions
+        id: commenter
+        uses: access-nri/actions/.github/actions/commenter-permission-check@main
+        with:
+          minimum-permission: write
```

Comment:
Could we add a comment to here to make sure future devs know that it's important not change this without understanding what it does

Reply:
Done in https://github.com/ACCESS-NRI/build-cd/pull/176/commits/4aca73c6712dbaba291cdc09ee9029bcadb60d5c